main/accounts/Account/index.js

```diff
@@ -345,6 +351,8 @@ class Account {
       if (this.signer) {
         const s = signers.get(this.signer)
         if (!s) return cb(new Error(`Cannot find signer for this account`))
+        const { tx, compatible } = signerCompatibility(rawTx, s.type)
+        if (tx === 'london' && !compatible) rawTx = londonToLegacy(rawTx)
```

Comment:
does the `Account` object need to know about this, or can we just push it down into the signer layer and have the individual signers responsible for converting if they need to?

Reply:
Signer level works, I chose the account level in case we need to check any additional account preferences or edit/update the request